headerFooter/style/MYHstatistikdel.css

```diff
@@ -0,0 +1,101 @@
+body {
+  margin: 0;
+  font-family: "Oswald", sans-serif;
+  width: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  display: grid;
+  grid-gap: 1rem;
+  grid-template-columns: repeat(auto-fit, minmax(20ch, 1fr));
+}
+
+@media screen and (max-width: 400px) {
+  .container {
+    display: grid;
+    grid-gap: 1rem;
+    grid-template-columns: repeat(auto-fit, minmax(20ch, 1fr));
+  }
+}
+
+.container div {
+  padding: 10px;
+  border-bottom: solid black;
+  border-bottom-width: 1px;
+}
+
+.header {
+  grid-column-start: 1;
+  grid-column-end: 4;
+  font-size: 30px;
+  font-weight: bold;
+}
+
+.content-large {
+  grid-row-start: 2;
+  grid-row-end: 3;
+  grid-column-start: 1;
+  grid-column-end: 3;
+  border-right: solid black;
+  border-right-width: 1px;
+}
+
+.content-small {
+  grid-row-start: 2;
+  grid-row-end: 3;
+  grid-column-start: 3;
+  grid-column-end: 4;
+}
+
+.mainText {
+  font-weight: bold;
+  font-size: 16px;
+}
+
+.links1 {
+  line-height: 1em;
+}
+
+.links3 {
+  position: relative;
+  bottom: 3em;
+  right: 20px;
+  list-style-type: none;
+  line-height: 3.5;
+  font-size: 12px;
+  font-weight: bold;
+  letter-spacing: 0.1em;
+  text-shadow: 1px rgba(3, 3, 3, 0.274), 1px rgba(14, 13, 13, 0.356);
+  padding: 2em;
+}
+
+.link3a {
+  border-bottom: solid black;
+  border-bottom-width: 1px;
+}
+
+.link3b {
+  border-bottom: solid black;
+  border-bottom-width: 1px;
+}
+
+.link3c {
+  border-bottom: solid black;
+  border-bottom-width: 1px;
+}
+
+.link3Img {

```

Comment:
men det här är en bild!

frågan är däremot om den faktiska bilden gör vara en css egenskap eller om det är något som man själv kodar in när man nyttjar klassen.

Reply:
Detta var klantigt, också ändrat. Ska i fortsättningen ha som utgångspunkt att ge logiska namn på klasserna från början.

Reply:
Har en fråga om du har tid. Hur skulle du gå till väga för att anpassa grid-boxarna så att de hamnar vertikalt för mobila enheter? Har testat olika sätt med media queries för att få en responsiv layout för olika skärmstorlekar, men det har inte fungerat hitintills. Fick det att fungera på första uppgiften, men inte här.

Tack!

Reply:
Uppdatering - det är löst!